nservicebus/messaging/routing_commands_core_[,6).partial.md

```diff
@@ -1 +1 @@
-Command routing can be configured via [MessageEndpointMappings](routing-extensibility.md#messageendpointmappings).
\ No newline at end of file
+Command routing can be configured via [MessageEndpointMappings](/nservicebus/messaging/routing-extensibility.md#messageendpointmappings).
```

Comment:
@SzymonPobiega why does this link use the absolute path while the other is using the relative path although they are both at the same location?

Reply:
My experimenting. Will change it back to relative.